libmount/src/context_mount.c

```diff
@@ -1414,8 +1420,16 @@ int mnt_context_get_mount_excode(
 		/*
 		 * /sbin/mount.<type> called, return status
 		 */
-		if (rc == -MNT_ERR_APPLYFLAGS && buf)
-			snprintf(buf, bufsz, _("WARNING: failed to apply propagation flags"));
+		if (buf) {
+			switch (rc) {
+			case -MNT_ERR_APPLYFLAGS:
+				snprintf(buf, bufsz, _("WARNING: failed to apply propagation flags"));
+				break;
+			case -MNT_ERR_EXEC:
+				snprintf(buf, bufsz, ("failed to execute %s"), cxt->helper);
```

Comment:
I suggest to use `_(` here and bellow  to show this string localized. There is the "failed to execute %s" string in *.po files.

Reply:
Oh, thanks. It's typo, there should be _() rather than () ;-)